css/style.css

```diff
@@ -77,3 +82,188 @@ a {
   display: flex;
   justify-content: space-between;
 }
+
+.social__icons li img {
+  width: 19.99px;
+  height: 19.99px;
+}
+
+.action__button {
+  margin-bottom: 20px;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+/* work section */
+#work .work-container {
+  display: grid;
+  row-gap: 20px;
+  grid-template-columns: 327px;
+  grid-template-rows: 170px 474px 474px 474px 474px 474px 474px;
+  justify-content: center;
+}
+
+#work .work-container .work-title {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  color: #172b4d;
+  text-align: center;
+  text-transform: capitalize;
+  width: 327px;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  background-image: linear-gradient(#36b37e, #36b37e);
+  background-size: 48px 4px;
+  background-position: center bottom;
+  background-repeat: no-repeat;
+  margin-bottom: 50px;
+}
+
+#work .work-container .work-card {
+  border-radius: 8px;
+  border: 1px solid #d0d9d4;
+}
+
+.work-card .work-img {
+  height: 252px;
+  width: 325px;
+}
+
+.work-card .text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 14px;
+}
+
+.project-title {
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 32px;
+  color: #3a4a42;
+  text-align: center;
+}
+
+.text .tag {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin: 20px 0;
+}
+
+.text .tag li {
+  padding: 8px 12px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  list-style: none;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+}
+
+button {
+  background: #36b37f;
+  border-radius: 4px;
+  padding: 12px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  border: none;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  height: 333px;
+  margin: 60px 0;
+  padding: 0 25px;
+  background-image: url("../images/about-me-top.svg"), url("../images/about-me-bottom.svg");
+  background-repeat: no-repeat;
+  background-position: top right, bottom left;
+}
```

Comment:
- [ ] As mentioned in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) that you must stick to the design as much as possible (e.g., font, colors, images, text, margins)

- [ ] Please make sure that your design is exactly the same as the one in [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). according to the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) as Figma design is very detailed.

Figma design 👇 
![bg22](https://user-images.githubusercontent.com/78251927/205117622-9963a1e6-ada9-4a2d-a078-1649add6ab45.PNG)

Your design 👇 
![bg11](https://user-images.githubusercontent.com/78251927/205117599-eaf950cc-d3a8-49e2-9d9d-fa348cd9488f.PNG)

- [ ] Please let us remove the (left and right) margins surrounding the background image as shown in the diagram above ☝️

Reply:
No, that's all 😊  👍

Reply:
ok I have fixed the background ... Can you review one more time ?

Reply:
Hi, @roniy68, you have to submit your project in your dashboard for re-reviews